src/NumericalAlgorithms/Interpolation/InterpolationTargetReceiveVars.hpp

```diff
@@ -249,21 +273,22 @@ struct InterpolationTargetReceiveVars {
       const std::vector<db::item_type<::Tags::Variables<
           typename InterpolationTargetTag::vars_to_interpolate_to_target>>>&
           vars_src,
-      const std::vector<std::vector<size_t>>& global_offsets) noexcept {
-    db::mutate<
-        Tags::IndicesOfFilledInterpPoints,
-        ::Tags::Variables<
-            typename InterpolationTargetTag::vars_to_interpolate_to_target>>(
+      const std::vector<std::vector<size_t>>& global_offsets,
+      const typename Metavariables::temporal_id::type& temporal_id) noexcept {
+    db::mutate<Tags::IndicesOfFilledInterpPoints<Metavariables>,
+               Tags::InterpolatedVars<InterpolationTargetTag, Metavariables>>(
         make_not_null(&box),
         [
-          &vars_src, &global_offsets
-        ](const gsl::not_null<db::item_type<Tags::IndicesOfFilledInterpPoints>*>
+          &temporal_id, &vars_src, &global_offsets
+        ](const gsl::not_null<
+              db::item_type<Tags::IndicesOfFilledInterpPoints<Metavariables>>*>
```

Comment:
please use the explicit type instead of item_type everywhere

Reply:
Just for my information:  Is the only place where you should use item_type  a place where you don't know the explicit type?

Reply:
Why not:
```cpp
    [](const gsl::not_null<Variables<vars_tags>*> vars,
       const Strahlkorper<Frame::Inertial>& strahlkorper,
       const FastFlow& fast_flow) noexcept
```
this is shorter, and less work from the compiler

Reply:
Ah, that is in FindApparentHorizon.hpp. Missed it before. Now made that change too.